modules/core/shared/src/test/scala/freestyle/tagless/StacksafeTests.scala

```diff
@@ -263,28 +263,23 @@ class TaglessStacksafeTests extends WordSpec with Matchers {
 
   }
 
-  "Tagless final algebras" should {
+  "A StackSafe Tagless algebra should be able to " when {
+    "mapK its Handler to another Handler of the StackSafe" in {
+      import cats.instances.list._
+      @tagless @stacksafe trait X {
+        def a: FS[Int]
+      }
+      object Y extends X.Handler[Option] {
+        def a: Option[Int] = Some(42)
+      }
+      val nt: Option ~> List = λ[Option ~> List](_.toList)
+      implicit val hh: X.Handler[List] = Y.mapK(nt)
```

Comment:
@loostro Per our Gitter discussion, ¿would this be enough?

Reply:
@diesalbla yeah, this example is exacly what I wanted intially to achieve

Reply:
@loostro can you share the example where you want `X.Handler` not to be `X` ?

Reply:
@peterneyens I could not get `implicit val logHandlerId: LoggingM.Handler[Id]` to work, after some struggle I found that if I change the type to `implicit val logHandlerId: LoggingM[Id]` it works (see gist), but it feels like a workaround - everywhere in the docs there is Handlers 
https://gist.github.com/loostro/f7b88dfb7dffbabd96a49e37307f262f

Reply:
If you are just using `@tagless` there is no benefit of using `Handler` as far as I see.

I would always write:
```scala
implicit val summerHandler: Summer[Id] = ...

implicit val logHandlerOption: LoggingM[Option] = ...
```

`Handler` is mostly useful if you want to use an `@tagless` algebra together with `@free` algebras, because it gives you an interpreter for `Foo` (tagless) and `Foo.StackSafe` (free). But I don't think that is a use case that comes up very often.

Reply:
It's fine, as it works without the `Handler`, but then it should be documented somehow.
On the other hand, it does not hurt to add mapK to Handler?